org.eclipse.m2e.discovery.oss/src/main/resources-filtered/connectors.xml

```diff
@@ -282,6 +282,26 @@ Finally, m2e-wtp helps you convert your legacy Eclipse projects to Maven, by tra
         <url>https://github.com/tesla/m2eclipse-subversive</url>
       </overview>
     </catalogItem>
+    
+    <catalogItem>
+        <categoryId>org.eclipse.m2e.discovery.category.scm</categoryId>
+        <description>Subversive (3.0+) SVN Integration for the M2E Project</description>
+        <m2e-versions>1.6</m2e-versions>
+        <groupId>extras</groupId>
+        <id>org.eclipse.m2e.discovery.lifecyclemapping.m2e-subversive</id>
+        <kind>extras</kind>
+        <license>Eclipse Public License - v1.0</license>
+        <name>m2e-subversive</name>
+        <provider>Eclipse.org</provider>
+        <p2>
+            <repositoryUrl>http://download.eclipse.org/technology/subversive/3.0/update-site/</repositoryUrl>
+            <iuId>org.eclipse.team.svn.m2e.feature.group</iuId>
+      </p2>
+      <overview>
+        <summary>Integration with M2E http://www.eclipse.org/m2e Installation is possible only if M2E is already installed or installed in one step with Eclipse Subversive.</summary>
```

Comment:
This summary is only relevant when installing manually from the subversive update site.

Use `<summary>Maven SCM Handler for Subversive</summary>` to be consistent with the other subversive entry